test/Microsoft.AspNetCore.Mvc.TestDiagnosticListener.Sources/project.json

```diff
@@ -3,10 +3,13 @@
   "shared": "**/*.cs",
   "frameworks": {
     "net451": {},
-    "dotnet5.4": {
+    "netstandard1.3": {
```

Comment:
Why does this project need anything but `net451` in `frameworks`? We don't build a package for this project and ignore the compiled DLL is always ignored.